#data
input_size: 14 #17
n_frames: 4
output_size: 3

#training
lr: 0.001
epochs: 20000
batch_size: 128
load_dir: /home/alfredo/canopies/code/CanopiesSimulatorROS/workspace/src/imitation_learning/data
save_dir: ./params
task: grasp

N_gaussians: 25
#